README.md

```diff
@@ -138,6 +138,7 @@ googleAnalytics = "" # Enable Google Analytics by entering your tracking id
 [[Params.widgets.social.custom]]
   title = "My Home Page"
   url = "https://example.com"
+  rel = "me" # Optional. The default is "noopener noreferrer". Set to false to remove the rel attribute
```

Comment:
```suggestion
  rel = "noopener noreferrer" # Set to false to remove the rel attribute
```

**README example:**

* Move `rel` to the YouTube example. "My Home Page" example should be minimal for now (title & url).
* Use `"noopener noreferrer"` directly in the value.
* Technically, not optional. Unlike the icon, it is always present by default, even if omitted in the config. The default value + comment with `false` because there are more examples in the documentation.

Reply:
Somehow I had missed that comment. I pushed the requested changes.

Reply:
The problem is still present, see suggested change. Remove "This is the default value" comment.

Reply:
Done